### ISSUE TITLE
0KB darwin.iso

### ISSUE THREAD

**Issue by umonaca** [paolo-projects/auto-unlocker#41]
VMWare Workstation Pro 16.1.0 build-17198959
OS: Windows 10

After running unlocker.exe, I can only get a 0KB darwin.iso.
But I get 3316KB from the old project gettools.exe here https://github.com/paolo-projects/unlocker

**Comment by giorgio79** [comment]
Seeing this issue as well. For me, this issue results in the Apple logo displaying on startup without the loading progress indicator.

**Comment by donovan-middleton** [comment]
Same here...  0KB file!

**Comment by umonaca** [comment]
You can always download the VMWare tools from their site.

**Comment by Huberdoggy** [comment]
I fixed this issue by manually extracting the copying over the darwin.iso from the tar archive which you’ll find in %tmp% on Windows after running the patch. Overwrite the 0KB darwin with the one from this directory and you should be good.

**Comment by paolo-projects** [comment]
Yes there's an issue with libarchive not extracting the zip file. I'm already working on a fix using libzip

**Comment by Huberdoggy** [comment]
> Yes there's an issue with libarchive not extracting the zip file. I'm already working on a fix using libzip

**Comment by Huberdoggy** [comment]
> Yes there's an issue with libarchive not extracting the zip file. I'm already working on a fix using libzip

It’s a great program @paolo-projects. I used the classic version with the additional command switches prior to using the ‘one click” auto-unlocker. You did a great job on both. At one point, I had to inspect your loop code because I accidentally overwrote the backup folder on a company PC and wanted to unpatch it so i was able to re-create the structure of the ‘backup’ folder by inspecting your source comments and copying fresh VMware files over from a colleague’s PC haha. Keep up the good work sir

**Comment by ling00100** [comment]
Error: cannot extract from zip file ，How do we solve this

**Comment by ling00100** [comment]
Error: cannot extract from zip file，Error: cannot extract from zip file
